integrations/huggingface-transformers/tutorials/sparse-transfer-learning-bert.md

```diff
@@ -0,0 +1,343 @@
+  --model_name_or_path zoo:nlp/masked_language_modeling/obert-base/pytorch/huggingface/wikipedia_bookcorpus/pruned90-none \
+  --distill_teacher zoo:nlp/document_classification/obert-base/pytorch/huggingface/imdb/base-none \
+  --recipe zoo:nlp/document_classification/obert-base/pytorch/huggingface/imdb/pruned90_quant-none \
+  --dataset_name imdb \
+  --do_train --do_eval --validation_ratio 0.1 --fp16 \
+  --evaluation_strategy steps --eval_steps 100 --logging_steps 100 --logging_first_step \
+  --per_device_train_batch_size 8 --per_device_eval_batch_size 32 --gradient_accumulation_steps 5 \
+  --save_strategy steps --save_steps 100 --save_total_limit 1
+  --preprocessing_num_workers 6 --max_seq_length 512 \
+  --seed 31222 \
+```
+
+- Text Classification: Sentiment Analysis (SST2)
+```bash
+sparseml.transformers.train.text_classification \
+  --output_dir sparse_quantized_bert-text_classification_sst2 \
+  --model_name_or_path zoo:nlp/masked_language_modeling/obert-base/pytorch/huggingface/wikipedia_bookcorpus/pruned90-none \
+  --recipe zoo:nlp/sentiment_analysis/obert-base/pytorch/huggingface/sst2/pruned90_quant-none \
+  --distill_teacher zoo:nlp/sentiment_analysis/obert-base/pytorch/huggingface/sst2/base-none \
+  --task_name sst2 \
+  --max_seq_length 128 --per_device_train_batch_size 32 --per_device_eval_batch_size 32 --preprocessing_num_workers 6 \
+  --do_train --do_eval --evaluation_strategy epoch --fp16  \
+  --save_strategy epoch --save_total_limit 1
+```
+
+- Token Classifcation: NER (Conll2003)
+```bash
+sparseml.transformers.train.token_classification \
+  --output_dir sparse_bert-token_classification_connl2003 \
+  --model_name_or_path zoo:nlp/masked_language_modeling/obert-base/pytorch/huggingface/wikipedia_bookcorpus/pruned90-none \
+  --distill_teacher zoo:nlp/token_classification/obert-base/pytorch/huggingface/conll2003/base-none \
+  --recipe zoo:nlp/token_classification/obert-base/pytorch/huggingface/conll2003/pruned90_quant-none \
+  --dataset_name conll2003 \
+  --per_device_train_batch_size 32 --per_device_eval_batch_size 32 --preprocessing_num_workers 6 \
+  --do_train --do_eval --evaluation_strategy epoch --fp16 --seed 29204  \
+  --save_strategy epoch --save_total_limit 1 
+```
+
+Check out the use case guides for more details on each task, including using a custom dataset and task specific arguments:
+- [Sentiment Analysis](sentiment-analysis/sentiment-analysis-cli.md)
+- [Text Classification](text-classification/text-classification-cli.md)
+- [Token Classification](token-classification/token-classification-cli.md)
+- [Question Answering](question-answering/question-answering-cli.md)
+
+## Wrap-Up
+
+Sparse transfer learning is just like typical fine-tuning - making it easy to create an inference-optimized 
+NLP model trained on your dataset!
+
+Checkout [DeepSparse](https://github.com/neuralmagic/deepsparse) for more details on reaching GPU-class performance by
+deploying the spars emodels on CPUs!
```

Comment:
Check out DeepSparse for more details on reaching GPU-class performance by deploying the sparse models on CPUs!